### PR TITLE
Feature #857 - Erc20 tokens support

### LINKED ISSUE #857: List DAI and USDT stable coin in Neuma Wallet
Goal: to list stable coins DAI and USDT in Neuma Wallet

From Will: 

> We should provide a list of ERC20 tokens and their contract addresses when instantiating an `EthereumConnection` instance.

> you can just use the same EthereumConnection as with ETH.
When creating an instance you pass EthereumConnectionOptions: https://github.com/iov-one/iov-core/blob/master/packages/iov-ethereum/src/ethereumconnection.ts#L130-L141
In there you can specify a map of ERC20 token tickers to options (covering eg contract address, decimals etc): https://github.com/iov-one/iov-core/blob/master/packages/iov-ethereum/src/erc20.ts#L25
From there you should be able to use the connection methods relating to ERC20 tokens. Check the tests for example usage. Eg getting an account with ERC20 tokens: https://github.com/iov-one/iov-core/blob/master/packages/iov-ethereum/src/ethereumconnection.spec.ts#L302-L314
Or sending ERC20 tokens: https://github.com/iov-one/iov-core/blob/master/packages/iov-ethereum/src/ethereumconnection.spec.ts#L596-L644

- [x] Add ERC20 coins support
- [x] Implement Unit testing

### PR DESCRIPTION
**Pull request by albertandrejev**
Closes #857.

### CODE REVIEW

File: packages/bierzo-wallet/src/utils/tokens.ts [hunk 1]
```diff
@@ -60,3 +62,21 @@ export async function chainAddressPairSortedMapping(
 
   return chainAddresses;
 }
+
+export function getErc20TokensConfig(options?: ConfigEthereumOptions): Erc20TokensMap | undefined {
+  if (!options) return undefined;
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/utils/tokens.ts]: To simplify the signature of this function, I'd make `options` non-optional. Then just call it conditionally:

```ts
erc20Tokens: spec.ethereumOptions ? getErc20TokensConfig(spec.ethereumOptions) : undefined,
```

**Comment by albertandrejev** [comment]
please do not merge this to master until we will be ready. @annatorok

File: packages/bierzo-wallet/src/config/development.json [hunk 2]
```diff
@@ -39,7 +39,16 @@
...
         "name": "Ganache",
         "node": "http://localhost:8545",
-        "scraper": "http://localhost:8546/api"
+        "scraper": "http://localhost:8546/api",
+        "ethereumOptions": {
+          "erc20s": [
+            {
+              "contractAddress": "0xCb642A87923580b6F7D07D1471F93361196f2650",
+              "decimals": 18,
+              "symbol": "ASHETH"
```
[the remark this answers is not in the document]
Comment (webmaster128) [reply packages/bierzo-wallet/src/config/development.json]: See https://github.com/iov-one/iov-core/issues/1287 for this issue

[the remark this answers is not in the document]
Comment (webmaster128) [reply packages/bierzo-wallet/src/config/development.json]: Feel free to ignore this as long as you are not running into trouble with renaming ASH to ASHETH. However, I plan to fix this quickly this week.

[the remark this answers is not in the document]
Comment (albertandrejev) [reply packages/bierzo-wallet/src/config/development.json]: ok, good

File: packages/bierzo-wallet/src/store/tokens/index.unit.spec.ts [hunk 3]
```diff
@@ -48,6 +48,14 @@ withChainsDescribe("Tokens reducer", () => {
           tokenTicker: "ASH",
         },
       },
+      ASHETH: {
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/store/tokens/index.unit.spec.ts]: Now that #863 is merged, can you please rebase this to master and rename this token to the original name `ASHETH` -> `ASH`? Thanks

File: packages/bierzo-wallet/src/config/staging.json [hunk 4]
```diff
@@ -31,7 +31,29 @@
...
+              "contractAddress": "0xc778417e063141139fce010982780140aa0cd5ab",
+              "decimals": 18,
+              "symbol": "WETH",
+              "name": "Wrapped Ether"
+            },
+            {
+              "contractAddress": "0x1f9061B953bBa0E36BF50F21876132DcF276fC6e",
+              "decimals": 0,
+              "symbol": "ZEENUS",
+              "name": "Zeenus"
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/config/staging.json]: No issue but just FYI: the emoji is part of the token name in the contract: https://rinkeby.etherscan.io/token/0x1f9061B953bBa0E36BF50F21876132DcF276fC6e#readContract

But we can rename the name to "Zeenus" without the emoji locally, no problem. Only the symbol is important to associate amounts with tokens.

File: packages/bierzo-wallet/src/store/tokens/index.unit.spec.ts [hunk 5]
```diff
@@ -40,6 +40,22 @@ withChainsDescribe("Tokens reducer", () => {
...
+        token: {
+          fractionalDigits: 18,
+          tokenName: "Trash Token",
+          tokenTicker: "TRASH",
+        },
+      },
+      ASH: {
+        chainId: "ethereum-eip155-5777",
+        token: {
+          fractionalDigits: 18,
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/store/tokens/index.unit.spec.ts]: This must be 12, see https://github.com/iov-one/ethereum-smart-contracts/blob/master/contracts/AshToken.sol

File: packages/sanes-browser-extension/config/development.json [hunk 6]
```diff
@@ -25,7 +25,22 @@
         "chainId": "ethereum-eip155-5777",
         "name": "Ganache",
         "node": "http://localhost:8545",
-        "scraper": "http://localhost:8546/api"
+        "scraper": "http://localhost:8546/api",
+        "ethereumOptions": {
+          "erc20s": [
+            {
+              "contractAddress": "0xCb642A87923580b6F7D07D1471F93361196f2650",
+              "decimals": 18,
```
Comment (webmaster128) [inline packages/sanes-browser-extension/config/development.json]: 12 here as well

**Pull request merged**